### ISSUE TITLE
Does the process of creating a project have no timeout period?

### ISSUE THREAD

**Issue by zoujiaqing** [dlang/dub#874]
Does the process of creating a project have no timeout period?

I create app project, a few hours passed without success.

![dub_init_app_hunt](https://cloud.githubusercontent.com/assets/444820/15986259/09e444ca-3036-11e6-9d3b-f24eca95df0d.png)

**Comment by nilufanier** [comment]
```
$ dub init app --vverbose
Using dub registry url 'http://code.dlang.org/'
Refreshing local packages (refresh existing: true)...
Looking for local package map at C:\ProgramData\dub\packages\local-packages.json
Looking for local package map at C:\Users\admin\AppData\Roaming\dub\packages\local-packages.json
No package found in current working directory.

```

**Comment by zoujiaqing** [comment]
Yes, Just like  @nilufanier !

**Comment by Geod24** [comment]
Duplicate of https://github.com/dlang/dub/issues/1682 and most likely fixed by #1841 . Any chance you can test @zoujiaqing ?
